requirements.txt

```diff
@@ -1,4 +1,4 @@
---index-url https://arti.dev.cray.com:443/artifactory/api/pypi/pypi-remote/simple
+--index-url https://arti.hpc.amslabs.hpecorp.net:443/artifactory/api/pypi/pypi-remote/simple
```

Comment:
That line should just be removed, not changed.  No need to use our internal mirror from the algol60 jenkins server.  That is also a step towards letting anyone build the open-source project.

Reply:
Fixed.